testsuite/podman_runner/19_get_server_logs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -xe
+
+if [ $# -ne 1 ];
+then
+	echo "Usage: ${0} server_id"
+	echo "server_id is used for creating a unique folder"
+	exit -1
+fi
+
+src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
+server_id=${1}
+rm -rfv /tmp/test-all-in-one/server-logs/${server_id}
+mkdir -p /tmp/test-all-in-one/server-logs/${server_id}
+sudo -i podman exec uyuni-server-all-in-one-test bash -c "tar jcvf /tmp/server-logs/${server_id}/var.log.rhn.tar.bz2 /var/log/rhn/"
```

Comment:
We don't need those, are already included in the supportconfig.

Reply:
ok. I have removed it .